solver-project-url-job/base/cronjob.yaml

```
@@ -0,0 +1,92 @@
+---
+apiVersion: batch/v1beta1
+kind: CronJob
+metadata:
+  name: solver-project-url-job
+  labels:
+    app: thoth
+    component: solver-project-url-job
+spec:
+  schedule: "@daily"
+  successfulJobsHistoryLimit: 2
+  failedJobsHistoryLimit: 4
```

```suggestion
  successfulJobsHistoryLimit: 1
  failedJobsHistoryLimit: 1
```
Lets keep these just one